files/Dockerfile

```diff
@@ -18,7 +18,12 @@ RUN docker-php-source extract
 # string (up the the !) is the pattern of the lines that will be ignored.
 # Needed for apt-get build-dep call later in script
 #
-RUN sed -Ei '/.*partner/! s/^# (deb-src .*)/\1/g' /etc/apt/sources.list
+
+# PHP 8.0
+#RUN sed -Ei '/.*partner/! s/^# (deb-src .*)/\1/g' /etc/apt/sources.list
+
+# PHP 8.1+
+RUN sed -Ei '/.*partner/! s/^# (deb-src .*)/\1/g' /etc/apt/sources.list.d/debian.sources
```

Comment:
Is this line even needed? The comment on lines 17-19 says it is needed to enable some? repo so that `apt-get build-dep` called later in the script would work. However I don't see `apt-get build-dep` called anywhere. Hence my question if this sed magic with debian sources file is needed at all.

Reply:
The effort to determine that as well as the question surrounding netcat is outside of the scope of this PR and requires it's own investigation IMO

Reply:
If it is not needed, why try to fix it? Have you tried removing it and building the agent inside `dev-shell`? That should be enough to determine if it is needed or not.

Reply:
Tested this and confirmed that it does not appear to impact agent build or tests.
Removed in 9b172b52c8341e1cefa938d8378369d252850835

Reply:
@bduranleau-nr Thank you for checking this and removing dead code from the codebase!